packages/components/table/src/tcell.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
```

Comment:
I don't think you need to import React here. Also... should we rename this component to Td?